Makefile

```diff
@@ -523,15 +495,15 @@ PUBLISH_BINTRAY_REDHAT += \
 endif
 
 ifeq ($(TARGET_PLATFORM),win32)
-electron-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
+electron-installer-portable: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH)-portable.exe
```

Comment:
The change from `$(TARGET_PLATFORM)` to `win32` might be redundant, given that no other platform uses `.exe` files?
Or perhaps this change does make sense for consistency reasons? \*shrug\*